mbof/models.py

```diff
@@ -59,6 +59,7 @@ def __str__(self):
 @python_2_unicode_compatible
 class Event(models.Model):
     title = models.CharField(max_length=50, default='Untitled')
+    address = models.CharField(max_length=500, default='Undefined')
```

Comment:
I believe if we set default to None then we would also have to set the parameter for null=True. We want these fields to be required fields moving forward.